stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/ChatApi.kt

```diff
@@ -33,7 +33,7 @@ internal interface ChatApi {
         channelId: String,
         file: File,
         callback: ProgressCallback,
-    )
+    ): Call<String>
```

Comment:
Just realized we should annotate all functions in this class that return a `Call` with `@CheckResult` - could we add that on this PR?